wiki/Skinning/Interface/en.md

```diff
@@ -753,7 +752,7 @@ _`arrow-warning.png`_ (transparent; not included)
 
 **Note:**
 
-- This is used in multiplayer games, seen next to the player's score (on left) when the player tries to skip the intro of a map.
+- This is used in multiplayer games, seen next to the player's score (on the sides) when the player votes to skip the intro of a map.
```

Comment:
by `left` I meant the leaderboards/scoreboards while playing, which is on the left